website/docs/features/dapp-dev-hot-reloading.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Hot releading during dApp development
+---
+
+Taqueria plugins are able to define templates that can be used to generate project components such as contracts, tests, or operations. Templates are exposed via plugins and use the `create` task to create and register the templates output
+
+## Available Templates
+
+| Template           | < template >         | Plugin                      | Command                       | Description                                                                                                        |
```

Comment:
Should we have the lines in here that talk about the contract registry? Is it still relevant?

Reply:
It will be deprecated in the future that's why I removed it